File: 10 - Python/Exercícios/01 - Estrutura Sequencial/Exercício 09.py
#Exercício - Medidas

A = float(input("Digite a medida A: "))
B = float(input("Digite a medida B: "))
C = float(input("Digite a medida C: "))

areaQuadrado = A ** 2
areaTriangulo = (A * B) / 2
areaTrapezio = (A + B) / 2 * C

print("Área do quadrado = ", round(areaQuadrado,4))
print("Área do triângulo = ", round(areaTriangulo,4))
print("Área do trapézio = ", round(areaTrapezio,4))

File: 06 - C/Exercícios/01 - Estrutura Sequencial/Exercício 09.c
//Exercício - Medidas

#include <stdio.h>
#include <math.h>

int main() {

    double A, B, C, areaQuadrado, areaTriangulo, areaTrapezio;

    printf("Digite a medida A: ");
    scanf("%lf", &A);

    printf("Digite a medida B: ");
    scanf("%lf", &B);

    printf("Digite a medida C: ");
    scanf("%lf", &C);

    areaQuadrado = pow(A, 2.0);
    areaTriangulo = (A * B) / 2.0;
    areaTrapezio = (A + B) / 2.0 * C;

    printf("Area do quadrado = %.4lf\n", areaQuadrado);
    printf("Area do triangulo = %.4lf\n", areaTriangulo);
    printf("Area do trapezio = %.4lf\n", areaTrapezio);

    return 0;

}